src/app/demo/demo/demo.component.html

```diff
@@ -127,6 +128,38 @@
         </div>
       </div>
 
+      <div class="dp-inline" *ngIf="pickerMode === 'timeInline'">
+        <dp-time-select
+            name="timeInline"
+            #timeInline="ngModel"
+            [theme]="material ? 'dp-material' : ''"
+            [(ngModel)]="date"
+            (ngModelChange)="log($event)"
+            [minDate]="validationMinDate"
```

Comment:
You're right, I should put the `minDate` and `maxDate` validation only on the `DayTime` component and not on the `Time` component.